scripts/README.md

```diff
@@ -0,0 +1,53 @@
+
+NAB scripts directory
+---------------------
+
+This directory contains some useful utility functions.
+
+##### Plotting results
+
+Please see plot.py, particularly the comments at the end. This script uses
+[plot.ly](http://plot.ly) so you need a (free) account and API key from there.
+It can generate plotly graphs for the raw data, labels, and/or detections.
+
+We used this script to generate the plots in the paper.  You will need to modify
+some lines at the bottom of the script to plot specific NAB data and/or results
+files.
+
+
+##### Alternative data and results visualization
+
+There is currently a simple and somewhat hacky data visualizer available, useful
+in hand labeling datasets. To use it do the following:
+
+First generate the list of data files and result files:
+
+    cd /path/to/nab
+    ls -1 data/*/*.csv | grep data > scripts/data_file_paths.txt
+    ls -1 results/*/*/*.csv | grep results | grep -v test_results > scripts/results_file_paths.txt
+    cd scripts
+    ln -s ../data
+    ln -s ../results
```

Comment:
is this correct? you're missing 2nd argument (I don't know what it does w/ just 1)